### ISSUE TITLE
conda 4.4.0rc1: conda update causes crash in python.exe

### ISSUE THREAD

**Issue by thomco** [conda/conda#5896]
[python.exe.12340.dmp.zip](https://github.com/conda/conda/files/1261004/python.exe.12340.dmp.zip)
On Windows 10 - 64 bit:

```
(base) C:\Users\Thomas> conda update --all --dry-run
Solving environment: reactivate does not accept arguments
remainder_args: ('--all', '--dry-run')
```
The crash occurs while "solving environment" spinner is active.   It reproduces with every package I've attempted to update.  

The rest of the output "reactivate does not accept arguments..."   is printed after the crash.

The crash occurs in C:\Users\Thomas\Miniconda3\python.exe, not conda itself.  Dump attached.
This started as soon as I updated to the rc1 on canary.   I had not tested rc0, I was updating from 4.3.25

```
(base) C:\Users\Thomas>where conda
C:\Users\Thomas\Miniconda3\Library\bin\conda.bat
C:\Users\Thomas\Miniconda3\Scripts\conda.exe
```
```
(base) C:\Users\Thomas>conda info

     active environment : base
    active env location : C:\Users\Thomas\Miniconda3
            shell level : 1
       user config file : C:\Users\Thomas\.condarc
 populated config files :
          conda version : 4.4.0rc1
    conda-build version : not installed
         python version : 3.6.2.final.0
       base environment : C:\Users\Thomas\Miniconda3  (writable)
           channel URLs : https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/win-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
          package cache : C:\Users\Thomas\Miniconda3\pkgs
                          C:\Users\Thomas\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\Thomas\Miniconda3\envs
                          C:\Users\Thomas\AppData\Local\conda\conda\envs
                          C:\Users\Thomas\.conda\envs
               platform : win-64
             user-agent : conda/4.4.0rc1 requests/2.14.2 CPython/3.6.2 Windows/10 Windows/10.0.16251
          administrator : False
             netrc file : None
           offline mode : False
```

**Comment by thomco** [comment]
Two notes:
1. I wanted to provided additional debugging info, so ran the command with -vvv.  Unfortunately, the crash does not repro with any of the verbose flags.   (Let me know if you want me to provide that output anyway)
2. I am running on a pre-release version of Windows 10.  Insider Preview build 16251.1002

**Comment by thomco** [comment]
BTW, It seems I cannot revert to 4.3

```
(base) C:\Users\Thomas>conda install conda=4.3 --no-deps
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - conda=4.3
  - pyopenssl[version='>=16.2']
  - cryptography[version='>=1.3.4']
  - openssl=1.0.2
  - *[provides_features='vc=10']

Current channels:

  - https://repo.continuum.io/pkgs/free/win-64
  - https://repo.continuum.io/pkgs/free/noarch
  - https://repo.continuum.io/pkgs/r/win-64
  - https://repo.continuum.io/pkgs/r/noarch
  - https://repo.continuum.io/pkgs/pro/win-64
  - https://repo.continuum.io/pkgs/pro/noarch
  - https://repo.continuum.io/pkgs/msys2/win-64
  - https://repo.continuum.io/pkgs/msys2/noarch


reactivate does not accept arguments
remainder_args: ('conda', '4.3', '--no-deps')
```

**Comment by bheklilr** [comment]
This looks like a duplicate of an issue I opened a while back, #5849

**Comment by thomco** [comment]
@bheklilr I agree.  Probably a dup.  
Hopefully the crash dump can help them find the cause.

**Comment by mingwandroid** [comment]
I believe I have fixed this and would appreciate your feedback.

From Linux, macOS or MSYS2/Git-for-Windows/Cygwin:

```
CONDA_AUTO_UPDATE_CONDA=False conda install -c c3i_test2 pycosat
```

From Windows cmd.exe:

```
set CONDA_AUTO_UPDATE_CONDA=False
conda install -c c3i_test2 pycosat
```

**Comment by kalefranz** [comment]
Fixed with an update to pycosat 0.6.3.
